src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery extends Ball {
+    Lottery() {
+        super();
+    }
+
+    Lottery(String color, int number) {
+        super(color, number);
+    }
+
+    // below method which provides Ball (randomColor, randomIndex);
+    Lottery getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        Random randomInt = new Random();
+        int number = randomInt.nextInt(101);

```

Comment:
Magic number should be constant with proper naming

Reply:
ok, done,